src/nessus.yml

```diff
@@ -4,13 +4,7 @@
   become: yes
   become_method: sudo
   roles:
+    - role: more_ephemeral_ports
     - role: nessus
       vars:
         package_bucket: "{{ build_bucket }}"
-
-- hosts: all
-  name: Configure maximum number of ephemeral ports
```

Comment:
If we're consolidating blocks then I think the name for the (above) block should be updated to reflect that change.

Reply:
See commit fbd3023.